packages/typespec-ts/package.json

```diff
@@ -33,7 +33,7 @@
     "@types/fs-extra": "^9.0.13",
     "@typescript-eslint/eslint-plugin": "^5.58.0",
     "@typescript-eslint/parser": "^5.58.0",
-    "@typespec/http": "~0.44.0",
```

Comment:
why do we put it here ? I think it's already in line 61 of the peer dependencies ?

Reply:
Yeah, so I removed here because it is already in peer one.